test/standalone/Soil/conservation.jl

```diff
@@ -64,10 +104,25 @@ for FT in (Float32, Float64)
             ClimaLand.total_liq_water_vol_per_area!(total_water, soil, Y, p, t0)
             expected = ν / 2 * (cmax - cmin)
             @test all(parent(total_water) .≈ expected)
+
+            dY = similar(Y)
+            exp_tendency! = make_exp_tendency(soil)
+            exp_tendency!(dY, Y, p, t0)
+            @test dY.soil.∫Fwdt == p.soil.∫Swdz
+            @test all(
+                parent(p.soil.∫Swdz) .≈ parent(
+                    ClimaCore.Fields.zeros(domain.space.surface) .+
+                    FT(((cmax - cmin) * -1e-5)),
+                ),
+            )
```

Comment:
Can we run this test without using parent? You could take the absolute difference and check that the max is smaller than a threshold